pkg/util/containers/v2/metrics/collector_docker_windows.go

```diff
@@ -0,0 +1,59 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+//go:build docker && windows
+// +build docker,windows
+
+package metrics
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util"
+	"github.com/docker/docker/api/types"
+)
+
+func convertContainerStats(stats *types.Stats) *ContainerStats {
```

Comment:
Because this collector also works on Windows, I think you need to add it here: https://github.com/DataDog/datadog-agent/blob/1b8a06d6fc5e59f2834f7408659e2eb28f9a182e/pkg/autodiscovery/listeners/environment.go#L78

Reply:
Fixed: https://github.com/DataDog/datadog-agent/pull/9612/files#diff-fb673103ed71b2f0eb00bcdabb898950e408964c5ac68e4fb6af37eaa1d5ed0b